src/mupenplus/Config_mupenplus.cpp

```diff
@@ -187,6 +188,112 @@ bool Config_SetDefault()
 	return ConfigSaveSection("Video-GLideN64") == M64ERR_SUCCESS;
 }
 
+void Config_LoadCustomConfig()
+{
+	if (ConfigExternalGetParameter == nullptr)
+		return;
+	const char* value;
+	std::string ROMname = RSP.romname;
+	const char* pathName = ConfigGetSharedDataFilepath("GLideN64.custom.ini");
+	for (size_t pos = ROMname.find(' '); pos != std::string::npos; pos = ROMname.find(' ', pos))
+		ROMname.replace(pos, 1, "%20");
+	for (size_t pos = ROMname.find('\''); pos != std::string::npos; pos = ROMname.find('\'', pos))
+		ROMname.replace(pos, 1, "%27");
+	std::transform(ROMname.begin(), ROMname.end(), ROMname.begin(), ::toupper);
```

Comment:
I'm not sure of a better way to convert the spaces into "%20" and the 's into %27, perhaps there is a way?